_posts/2020-07-10-removing-negative-words.markdown

```diff
@@ -96,3 +104,20 @@ We have listened to the community. We have seen the trends. And we're committed
 This is not an isolated initiative. OmbuLabs is committed to sustaining an open, diverse and inclusive work environment as a whole. This is one facet of that environment, and an important one.
 
 There are several arguments, human and technical, to move away from negative terminology. I hope this article helps you not only understand why we're doing this, but to advocate change in your projects and help us change as a community.
+
+## Sources and References
+
+1. [Developers Debate Deleting ‘Master’ and ‘Slave’ Code Terminology](https://insights.dice.com/2020/06/16/developers-debate-deleting-master-slave-code-terminology/) by Nick Kolakowski o[Tech Confronts Its Use of the Labels ‘Master’ and ‘Slave’n [Dice](https://insights.dice.com/).
```

Comment:
Something strange is happening here, is this 2 different articles?

Reply:
No, that's the wrong article pasted in the wrong place. I'll fix it.